.core_files.yaml

```diff
@@ -58,6 +58,7 @@ base_platforms: &base_platforms
 # Extra components that trigger the full suite
 components: &components
```

Comment:
This list is a hodge-podge of various things.
At the very least, I think we should extract integrations which are in this list because other integrations have platforms with them but are not providing base entities to its own list.